influxdb/client.py

```diff
@@ -75,10 +77,12 @@ def __init__(self,
                  udp_port=4444,
                  proxies=None,
                  pool_size=10,
+                 path='',
                  ):
         """Construct a new InfluxDBClient object."""
         self.__host = host
         self.__port = int(port)
+        self.__path = path if not path or path[0] == '/' else '/' + path
```

Comment:
Since you're slicing the string here, perhaps it would make sense to first cast the input to a string (similar to the above port cast)?

Reply:
So you'e prefer something like:
```
_path = str(path)
self.__path = _path if not _path or _path[0] == '/' else '/' + _path
```
Any other suggestion how to make this look better?

Reply:
It dosen't have to be a oneliner

Reply:
Sure, my suggestion was no longer a one-liner either. What should it be like?